app/helpers/internationalisation_helper.rb

```diff
@@ -0,0 +1,30 @@
+module InternationalisationHelper
+
+  def t(*args)
```

Comment:
add documentation to this function, as it is not a usual thing to do. And future people will strangle in this.